lib/cipher/symmetric/schwaemm.ts

```
@@ -0,0 +1,182 @@
+    yearDesigned: 2019,
+    standardBody: 'NIST SP 800-232',
+}
+
+const ALZETTE_C = [0xB7E15162, 0xBF715880, 0x38B4DA56, 0x324E7738, 0xBB1185EB, 0x4F7C7B57]
+const DOMAIN_I_PADDED = 0x01, DOMAIN_I_COMPLETE = 0x02, DOMAIN_M_PADDED = 0x03, DOMAIN_M_COMPLETE = 0x04
+
+function u32(n: number): number { return n >>> 0 }
+function rotr(x: number, n: number): number { return u32((x >>> n) | (x << (32 - n))) }
+
+function alzette(x: number, y: number, c: number): [number, number] {
+    let x1 = u32(x + rotr(y, 24)); let y1 = u32(y ^ rotr(x1, 31));
```

ESLint

[error] 'x1' is never reassigned. Use 'const' instead.

(prefer-const)